### PR TITLE
Compatibility with Autofac v6 and project system update

### PR DESCRIPTION
**Pull request by alexmg**
- Update to Autofac `6.0.0-develop-01134` and fix breaking changes
- Bump package version to `6.0.0`
- Upgrade projects to SDK format
- Upgrade build system
- Update copyright headers
- Fix code style issues

### CODE REVIEW

**Comment by alistairjevans** [comment]
Afraid I'm going to have to leave this with @tillig, as I'm away now for a week.

**Comment by alexmg** [comment]
Well played @alistairjevans! 🤣

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Try StackOverflow! If you have a _usage question_ ("How do I...?" "Others have this working but I can't get it...") please try [StackOverflow](https://stackoverflow.com) and tag your question `autofac`. We, along with a great community, monitor those questions.
```
Comment (tillig) [inline ISSUE_TEMPLATE.md]: Nuke this. We added a `.github` repo that has all the common files like issue templates, CONTRIBUTING, and so on. GitHub will default to the ones in the common repo if they aren't present here.

File: test/Autofac.Integration.Wcf.Test/Autofac.Integration.Wcf.Test.csproj [hunk 2]
```diff
@@ -1,105 +1,42 @@
...
-    <AssemblyOriginatorKeyFile>..\..\Autofac.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkProfile />
-    <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\..\..\</SolutionDir>
-    <RestorePackages>true</RestorePackages>
-    <NuGetPackageImportStamp>
-    </NuGetPackageImportStamp>
+    <TargetFramework>net472</TargetFramework>
+    <NoWarn>$(NoWarn);CS1591;SA1602;SA1611;CA1812</NoWarn>
+    <IsPackable>false</IsPackable>
   </PropertyGroup>
```
Comment (tillig) [inline test/Autofac.Integration.Wcf.Test/Autofac.Integration.Wcf.Test.csproj]: This is sorta weird that they ended up in one-off `<PropertyGroup>` elements.

**Review by tillig — CHANGES_REQUESTED** [review]
Couple minor things. Nice work getting this thing wrangled!

**Pull request merged**